SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/CrossServerPossessionLockTest.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "CrossServerPossessionLockTest.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "GameFramework/PlayerController.h"
+#include "Net/UnrealNetwork.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "TestPossessionPawn.h"
+#include "TestPossessionPlayerController.h"
+
+/**
```

Comment:
The description does not match the test's behaviour.

Reply:
updated, Please take a look, thanks

Reply:
The setup section was better before, describing how the test was setup in terms of what and where the actors were (event if the description itself was incorrect).